### DISCUSSION TITLE
Multi Virtual Screens

### DISCUSSION THREAD

**Discussion by yahyatube** [ClassicOldSong/Apollo#42] [Q&A] [answered]
I would like to create a large number of Virtual Screens for each phone. Is this possible?

Currently only one person is allowed to connect to the virtual screen

**Comment by ClassicOldSong** [comment, marked as answer]
See Wiki: https://github.com/ClassicOldSong/Apollo/wiki/How-to-start-multiple-instances-of-Apollo

**Comment by yahyatube** [comment]
Thank you, the latest update is very wonderful
